e2e-tests/controller-spark/controller_spark_sql_validation.sh

```diff
@@ -88,15 +89,50 @@ function setup() {
   HOME_PATH=$1
   PARQUET_SUBDIR=$2
   SOURCE_FHIR_SERVER_URL='http://localhost:8091'
+  SINK_FHIR_SERVER_URL='http://localhost:8098'
   PIPELINE_CONTROLLER_URL='http://localhost:8090'
   THRIFTSERVER_URL='localhost:10001'
   if [[ $3 = "--use_docker_network" ]]; then
     SOURCE_FHIR_SERVER_URL='http://hapi-server:8080'
+    SINK_FHIR_SERVER_URL='http://sink-server:8080'
     PIPELINE_CONTROLLER_URL='http://pipeline-controller:8080'
     THRIFTSERVER_URL='spark:10000'
   fi
 }
 
+#######################################################################
+# Function queries fhir server and write results to json file
+# Globals:
+#   HOME_PATH
+#   PARQUET_SUBDIR
+# Arguments:
+#  server_url: flag to differentiate behavior between incremental and batch runs.
```

Comment:
This definition for `server_url` does not seem right.

Reply:
done